### PR TITLE
HoC 2023 - Add Dance Party AI skinny banners to code.org

### PR DESCRIPTION
**Pull request by kelbyhawn**
Add a Dance Party: AI Edition skinny banner to the following pages:
- Teach Page - [code.org/teach](http://code.org/teach) 
- AI landing page - [code.org/ai](http://code.org/ai) 
- How AI Works - [code.org/ai/how-ai-works](http://code.org/ai/how-ai-works) 
- AI 101 for Teachers - [code.org/ai/pl/101](http://code.org/ai/pl/101)

**Jira ticket:** [ACQ-993](https://codedotorg.atlassian.net/browse/ACQ-993)

----

## Desktop
Desktop

## Tablet
Tablet

## Mobile
Mobile

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/ai/index.haml [hunk 1]
```diff
@@ -60,15 +53,7 @@ theme: responsive_full_width
...
-%section#ai-pl-callout.bg-primary-light.blog-post-callout
-  .wrapper
-    %i{class: "#{icon_megaphone}"}
-    .text-wrapper
-      %h2.heading-lg=hoc_s(:ai_pl_callout_title)
-      %p
-        %strong=hoc_s(:ai_pl_callout_desc_02)
-      %p.body-three=hoc_s(:ai_pl_callout_desc_01)
-      %a.link-button{href: "/ai/pl/101"}=hoc_s(:signup_submit_explore)
+= view :dance_party_ai_skinny_banner
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/ai/index.haml:56]: Added the new banner here, and removed the AI 101 callout (`#ai-pl-callout`) and CSA AI skinny banner (`= view :csa_ai_skinny_banner`):

AI page

File: pegasus/sites.v3/code.org/public/ai/pl/101/index.haml [hunk 2]
```diff
@@ -28,6 +28,8 @@ theme: responsive_full_width
       =hoc_s(:ai_pl_callout_desc_01)
     = view :ai_101_videos
 
+= view :dance_party_ai_skinny_banner
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/ai/pl/101/index.haml:31]: Added the new banner here:

Screenshot 2023-11-21 at 10 52 54 AM

File: pegasus/sites.v3/code.org/public/teach.haml [hunk 3]
```diff
@@ -19,7 +19,7 @@ theme: responsive_full_width
       %img.rounded-corners.box-shadow-primary{src: "/images/teach-page-top.png", alt: "", style: "position: relative; width: 100%;"}
     .clear
 
-= view :ai_pl_skinny_banner
+= view :dance_party_ai_skinny_banner
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/teach.haml:22]: Added the new banner here:

Teach page

File: pegasus/sites.v3/code.org/public/teach.haml [hunk 4]
```diff
@@ -104,6 +104,8 @@ theme: responsive_full_width
       %a.link-button.secondary{href: CDO.studio_url("/catalog")}
         =hoc_s(:call_to_action_see_full_curriculum_catalog)
 
+= view :ai_pl_skinny_banner
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/teach.haml:107]: Moved this banner down here:

Screenshot 2023-11-21 at 10 49 53 AM

File: pegasus/sites.v3/code.org/public/ai/how-ai-works.haml [hunk 5]
```diff
@@ -32,6 +32,8 @@ theme: responsive_full_width
         =hoc_s(:call_to_action_explore_lessons)
     .clear
 
+= view :dance_party_ai_skinny_banner
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/ai/how-ai-works.haml:35]: Added the new banner here:

ai_how-ai-works

**Review by bethanyaconnor — APPROVED** [review]
LGTM!

**Pull request merged**